HW04/ihar_shkuryn/lib/mentor.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+class Mentor
+  attr_reader :name, :surname, :homework, :self
+  attr_accessor :notices, :all_homeworks, :homeworks_for_check
+
+  def initialize(data)
+    @name = data[:name]
+    @surname = data[:surname]
+    @subscriptions = []
+    @notices = []
+    @all_homeworks = []
+    @homeworks_for_check = []
+  end
+
+  def add_homework(data)
+    student = data[:student]
+    homework = Homework.new(data, self)
+    homework.message = "Student #{student.name} #{student.surname} got a new homework "\
+    "from #{name} #{surname} title: #{@title}"
+    all_homeworks.push(homework)
+    student.homeworks_todo.push(homework)
+    notice_handler(homework, student)
+    homework
+  end
+
+  def notice_handler(homework, student)
+    notice = Notification.new(homework.title)
+    student.notices.push(notice)
+  end
+
+  def subscribe_to!(student)
+    @subscriptions.push(student)
```

Comment:
```suggestion
    subscriptions.push(student)
```
add subscriptions to attr_reader